Backend-Challenges.md

```diff
@@ -71,13 +77,13 @@ In `Challenge_6.t.sol`, we want to get the token URI from the contract given a c
 
 ### Solidity libraries and openzeppelin
 
-In Solidity, you can import libraries that will make life easier. OpenZeppelin is a company that provides free standardized contract libraries to use, such as ERC721. These contracts are firmly audited, meaning they are generally secure to implement in your contracts.
+In Solidity, you can import libraries. Because upgradeability is hard. In the unforgiving environment for bugs plus time pressure for software development, libraries will make your life easier. OpenZeppelin is a company that provides free standardized contract libraries to use, such as ERC721. These contracts are firmly audited, meaning they are generally secure to implement in your contracts.
```

Comment:
This one doesn't really seem to work well, i'd suggest another try or use the below format

Solidity offers support to import libraries, both internal and external. More so than other languages, libraries play a key part in the development of smart contracts. This is because upgradability is quite a bit trickier, the environment is unforgiving for bugs, and then there's the all too common time pressure for software development. Put simply, libraries will make your life easier. 
Openzeppelin is a company that provides free, standardized, and thoroughly tested/audited smart contract libraries to use, such as ERC721.